lib/node_modules/@stdlib/repl/lib/display_prompt.js

```diff
@@ -26,6 +26,11 @@ var repeat = require( '@stdlib/string/repeat' );
 var leadingWhitespaceRegExp = require( './regexp_leading_whitespace.js' );
 
 
+// VARIABLES //
+
+var BASE_PROMPT = '...: ';
```

Comment:
How would this work when a user provides a custom prompt? This assumes that the prompt is of the form `In [%d]:`, but that is not necessarily the case. See https://github.com/stdlib-js/stdlib/tree/develop/lib/node_modules/%40stdlib/repl#repl-options-.

Reply:
E.g., the prompt could be changed to `> ` or `$ ` or something else entirely. In which case, this implementation is making an assumption about the form of the prompt which is not guaranteed to be true.

Reply:
You are right, I messed up there. One solution can be, the prompt can just consist of spaces/dots the length of the input prompt, the lines would still align , there would be no `...:` in the prompt

Reply:
I think this returns us to whether a continuation prompt is necessary. For example, if we support SHIFT+ENTER or some other modifier, then

```
In [5]:
function func() {
	console.log( 'hello world' );
}
```

should be possible which achieves alignment, without requiring special prompt handling or the loss of terminal columns with unused whitespace. For the latter in particular, while some terminal applications are good in handling text overflow, others are not, thus making horizontal real estate something of a precious commodity. If we followed IPython and provided a continuation prompt which indents based on the parent prompt, we limit full use of all possible real estate.

Reply:
Makes sense.. I'll close this issue and PR👍